packages/modeling/src/operations/transforms/align.test.js

```diff
@@ -53,7 +53,7 @@ test('align: multiple objects ungrouped returns geometry aligned, different mode
     cube({ size: 4, center: [10, 10, 10] }),
     cube({ size: 2, center: [4, 4, 4] })
   ]
-  const aligned = align({ modes: ['center', 'min', 'max'], relativeTo: [30, 30, 30] }, original)
+  const aligned = align({ modes: ['center', 'min', 'max'], relativeTo: [30, 30, 30] }, ...original)
```

Comment:
why the change? shouldn't align() determine what to do with an array of objects versus a single object?

Reply:
This is related to the change in `align` behavior that was discussed in the original PR description. Basically I am proposing a change where each separate argument to `align` will be aligned as a group, but different top-level args are treated separately.

Previously, in V2, the behavior was to flatten everything:

`align({}, [a, b], [c, d]) === [align({}, a), align({}, b), align({}, c), align({}, d)]`
`align({}, [[a, b], [c, d]]) === [align({}, a), align({}, b), align({}, c), align({}, d)]`

Whereas with this PR I propose that:

`align({}, [a, b], [c, d]) === [align({}, [a, b]), align({}, [c, d])]`
`align({}, [[a, b], [c, d]]) === align({}, [[a, b], [c, d]])` (cannot be decomposed because the whole array is grouped)

This was probably one of the more debatable changes in this PR, but I included it based on my experience using `align` for designs with groups, and feedback in #1249. Following design principle number 4:  This gives more flexibility to the users, because users can change the grouping (multiple args, vs one array arg), they can select different alignment behavior. Whereas before users had less flexibility.

Reply:
That's good documentation for the V2 -> V3 migration, and probably worth saving somewhere.

OK. So, going back to the test. I now see the subtle difference in the V3 API, which changed the results of the test. Shouldn't the test be changed to verify the new results, rather then tweaking the input to '...original'? Or even better, change the test to pass in two parameters instead of the array, which is probably what most users will do.

All of this is probably just 'style' debates. Sorry.